tool/src/test/java/org/datacommons/tool/GenMcfTest.java

```diff
@@ -85,6 +89,7 @@ public void GenMcfTest() throws IOException {
       }
       argsList.add("--resolution=FULL");
       argsList.add("--output-dir=" + Paths.get(testFolder.getRoot().getPath(), testName));
+      argsList.add("--summary-report");
```

Comment:
Unlike absl, for these flags I think we need to specify true or false, otherwise it seemed to toggle the default...

Reply:
done